react-charts/stack-chart/step7.md

```diff
@@ -1,56 +1,37 @@
-Now we'll add a legend to the `Chart` component...
+Change the stack chart theme color.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the `<Chart>` component and add a property to change the theme.</strong>
 
-2) Locate the code that looks like the following:
+

```

Comment:
As per area-chart changes, you can remove the "it should look like this" / and the code snippet.. see area-chart step 7